client/blocks/login/style.scss

```diff
@@ -47,7 +47,6 @@
 
 .login__form-action {
 	margin-top: 10px;
```

Comment:
It looks like you could also remove that line.

Reply:
Sounds great, thank you @stephanethomas.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/30511/commits/0f0b26c3c50c43679f807c75a5f8f89c890c1bc9